### ISSUE TITLE
Cannot Install with Carthage: "xcodebuild timed out while trying to read"

### ISSUE THREAD

**Issue by hainguyen86** [whitesmith/WSTagsField#4]
Hi @AgentWS ,
I'm trying install your framework by using Carthage 
`github "whitesmith/WSTagsField"`

Then, the terminal said:

> xcodebuild timed out while trying to read WSTagsFieldExample.xcodeproj 😭

Please fix this asap

Thank you very much!

**Comment by ricardopereira** [comment]
Well, this is working: `xcodebuild -project './WSTagsField.xcodeproj' -scheme 'WSTagsField' -destination 'platform=iOS Simulator,OS=latest,name=iPhone 5s' build`

```
=== BUILD TARGET WSTagsField OF PROJECT WSTagsField WITH CONFIGURATION Debug ===

Check dependencies
...

** BUILD SUCCEEDED **
```

**Comment by ricardopereira** [comment]
I'm having the same issue with `carthage version 0.17.2` and the log is empty.

**Comment by ricardopereira** [comment]
Fixed on [v1.2](https://github.com/whitesmith/WSTagsField/releases/tag/1.2).

**Comment by hainguyen86** [comment]
thanks @ricardopereira
